pkg/network/network.go

```diff
@@ -377,7 +385,32 @@ func cnaoKNMStateObjects(operandNamespace string) ([]*unstructured.Unstructured,
 
 	convertedObjects := []*unstructured.Unstructured{}
 	for _, object := range objects {
-		err := addTypeInformationToObject(object)
+		err := addTypeInformationToObject(scheme, object)
+		if err != nil {
+			return nil, err
+		}
+		convertedObject, err := runtime.DefaultUnstructuredConverter.ToUnstructured(object)
```

Comment:
is it a must ?
until now we didnt have to do such thing ?

Reply:
Will create a helper.